diagnostic_rt_qpcr/operation_types/run_qpcr/protocol.rb

```diff
@@ -155,13 +154,13 @@ def get_available_thermocyclers
       note 'Please check which thermocyclers are currently available'
       thermocyclers.each_with_index do |thermo|
         select([available_key, 'unavailable'],
-               var: thermo['name'].to_s,
+               var: thermo['name'].to_sym,
```

Comment:
Can we avoid converting these to symbols? It might make sense to make `find_thermocyclers` return a HashWithIdifferentAccess

Reply:
I am confused as to how a HashWithIndifferentAccess would be used in this case.  All the hashes are currently determined by either Parameters or by the "select" method in show....      Will discuss at our 9:30

Reply:
OK, you're probably right.